docs/documentarians.rst

```diff
@@ -0,0 +1,26 @@
+Documentarians
+==============
+
+When we :doc:`created </origin-story>` Write the Docs,
+we felt there was a missing term for someone who cares about documentation.
+It shouldn't be a job title,
+but an independent identity.
+We started using the word *documentarian* to describe this type of person:
+
+**A documentarian is someone who cares about the value of documentation and communication in the software industry, regardless of job title.**
```

Comment:
Do you prefer "cares about the value of documentation" to "cares about documentation" or even "values documentation"?